react/components/EXPERIMENTAL_Charts/LineChart/index.tsx

```diff
@@ -11,46 +11,49 @@ import {
   TooltipFormatter,
 } from 'recharts'
 import PropTypes from 'prop-types'
-import { colors, defaultProps, tooltipProps } from './constants'
-import getDefaultProps from '../helpers'
+import { tooltipProps } from './constants'
+import { getChartDefaultProps } from '../helpers'
+import getLineDefaultProps from './helpers'
+import uuid from 'uuid'
+import { colors, commonDefaultProps} from '../commonProps'
 
 interface Props {
-  data: any,
-  dataKeys: string[],
-  xAxisKey: string,
-  schema: ChartProps,
-  formatter: TooltipFormatter
+  formatter: TooltipFormatter,
+  lineProps: LineProps
 }
 
-const renderLine = (key, color) => (
+const renderLine = (lineConfigs, key, color) =>(
   <Line
-    key={key}
+    key={`${key}--${uuid()}`}
     dataKey={key}
     stroke={color}
-    {...defaultProps}
+    {...lineConfigs}
   />
 )
 
-const LineChart: FC<Props> = ({
+const LineChart: FC<Props & BaseChartProps> = ({
   data,
   dataKeys,
   xAxisKey,
   schema,
-  formatter
+  formatter,
+  lineProps
 }) => {
-  const { configs } = getDefaultProps(schema); 
+  const { configs } = getChartDefaultProps(commonDefaultProps, schema)
+  console.log(configs)
+  const { lineConfigs } = getLineDefaultProps(lineProps)
 
   return (
     <ResponsiveContainer {...configs.container}>
       <LineChartBase data={data}>
-        <XAxis dataKey={xAxisKey} {...configs.axis} />
-        <YAxis {...configs.axis}/>
+        <XAxis dataKey={xAxisKey} {...configs.xAxis}/>
```

Comment:
Try to use an automatic linter to keep consistency.
```suggestion
        <XAxis dataKey={xAxisKey} {...configs.xAxis} />
```